### PR TITLE
fp7/update app_comprehensive report

### PR DESCRIPTION
**Pull request by Fraybeke**

### CODE REVIEW

File: src/components/pages/reports/app_comprehensive_report.php [hunk 1]
```diff
@@ -947,21 +949,20 @@ function drawSecurityChart() {
         chart.draw(data, options);
       }
     
-    //*******************************************************/
+    //****************************************************** */
       function drawComponentCount() {
         var data = google.visualization.arrayToDataTable([
-          ['App name', 'Issue Count', 'Total Issue Count'],
+          ['App name', 'OSS Count', 'Commercial Count'],
```
Comment (CZend371) [inline src/components/pages/reports/app_comprehensive_report.php:955]: Don't we need `App Version` as well?

File: src/components/pages/reports/app_comprehensive_report.php [hunk 2]
```diff
@@ -947,21 +949,20 @@ function drawSecurityChart() {
...
-          as num_issue, SUM(issue_count) as total_issue_count 
-          FROM apps_components 
+          $query = $db->query("SELECT app_name, app_version, SUM(CASE WHEN license NOT LIKE '%Commercial%' THEN 1 ELSE 0 END) as oss_count, SUM(CASE WHEN license LIKE '%Commercial%' THEN 1 ELSE 0 END) as commercial_count, COUNT(license) as total
+          FROM apps_components
           GROUP BY app_name;");
           while ($query_row = $query->fetch_assoc()) {
               $app_name=$query_row['app_name'];
-              $num_issue=$query_row['num_issue'];
-              $total_issue_count=$query_row['total_issue_count'];
+              $oss_count=$query_row['oss_count'];
```
Comment (CZend371) [inline src/components/pages/reports/app_comprehensive_report.php:962]: Same here.

File: src/components/pages/reports/app_comprehensive_report.php [hunk 3]
```diff
@@ -947,21 +949,20 @@ function drawSecurityChart() {
...
+              $commercial_count=$query_row['commercial_count'];
            ?>
-           ['<?php echo $app_name;?>',<?php echo $num_issue;?>,<?php echo $total_issue_count;?>],   
+           ['<?php echo $app_name;?>',<?php echo $oss_count;?>,<?php echo $commercial_count;?>],
```
Comment (CZend371) [inline src/components/pages/reports/app_comprehensive_report.php:965]: And here.

File: src/components/pages/reports/app_comprehensive_report.php (hunk 1 above)
Comment (Fraybeke) [reply src/components/pages/reports/app_comprehensive_report.php:955]: on the graph?

Comment (CZend371) [reply src/components/pages/reports/app_comprehensive_report.php:955]: Yea. Since it's in my query.

**Review by CZend371 — APPROVED** [review]

**Pull request merged**